build/build-manual.js

```diff
@@ -6,7 +6,9 @@ var templates = require('./templates');
 var Promise = require('promise');
 
 module.exports = function build(grunt, options, commons, callback) {
-	options.getFiles = options.hasOwnProperty('getFiles') ? options.getFiles : true;
+	options.getFiles = options.hasOwnProperty('getFiles')
```

Comment:
yikes - really?

Reply:
Yes, Prettier does everything it can to fit everything on a single line, but when it can't, it has to add newlines.